src/internal/metadata/metadataReading.ts

```diff
@@ -22,7 +22,7 @@ export function metadataFromInstance(h: any): CommandHandlerMetadata | EventHand
     if (isEventHandlerMetadata(h)) {
         md = addName(h);
     } else if (isCommandHandlerMetadata(h)) {
-        md = addName(h);
+        md = addName(h); // JESS: this seems mighty incomplete to me.
```

Comment:
A note for later?

Reply:
yeah, I suspect that whole case doesn't actually work. which is not related to this PR.